code/modules/mob/living/simple_animal/hostile/retaliate/dolphin.dm

```diff
@@ -0,0 +1,48 @@
+/mob/living/simple_animal/hostile/retaliate/dolphin
+	name = "space dolphin"
+	desc = "That's LIEUTENANT commander to you!" //sounds like a reference but is not
```

Comment:
First of all, don't leave comments like that, and second can we please have at least half of a normal description?